packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -654,13 +648,11 @@ const PostsPage = ({fullPost, postPreload, eagerPostComments, refetch, classes}:
     </AnalyticsContext>
   </>;
 
-  const maybeWelcomeBoxProps = forumSelect(welcomeBoxes);
-  const welcomeBoxProps = welcomeBoxABTestGroup === "welcomeBox" && !currentUser && maybeWelcomeBoxProps;
-  const welcomeBox = welcomeBoxProps
-    ? <div className={classes.welcomeBox}>
-        <WelcomeBox {...welcomeBoxProps} />
-      </div>
-    : null;
+  const welcomeBox = (
+    <ForumNoSSR>
+      <WelcomeBox />
+    </ForumNoSSR>
+  );
```

Comment:
I talked about it with Jim on a call, the layout shift actually isn't that bad because it doesn't need to load any data so it happens fairly instantly. cc @b0b3rt